server/accounts.go

```diff
@@ -508,10 +508,6 @@ func (a *Account) TrackServiceExportWithSampling(service, results string, sampli
 		return ErrBadPublishSubject
 	}
 
-	if a.srv == nil || !a.srv.eventsEnabled() {
-		return ErrNoSysAccount
-	}
```

Comment:
This check had to be moved to be done earlier when processing the config at: https://github.com/nats-io/nats-server/pull/1130/files#diff-7b256a4f472c7d4b9e1931f2f1073b66R401-R409